packages/protocol/index.js

```diff
@@ -0,0 +1,20 @@
+const AirnodeArtifact = require('./artifacts/contracts/Airnode.sol/Airnode.json');
+const ConvenienceArtifact = require('./artifacts/contracts/Convenience.sol/Convenience.json');
+
+const networkChainIds = { ropsten: 3, rinkeby: 4 };
```

Comment:
I initially exported all deployments present in the directory, but I guess hardcoding these is a better idea. This way it will error if the deployments can't be found.